torch/csrc/autograd/profiler_legacy.h

```diff
@@ -291,6 +292,23 @@ struct TORCH_API LegacyEvent {
     scope_ = scope;
   }
 
+  const std::unordered_map<std::string, c10::IValue>& extraArgs() const {
+    return extra_args_;
+  }
+
+  void setExtra(const at::RecordFunction& fn) {
+    if(fn.inputs().size() != 0) {
+      saveExtraArgs(extra_args_, fn);
+    }
+  }
+
+  uint64_t flops(void) {
+    if(extra_args_.size() == 0) {
```

Comment:
nit: space, and everywhere else

Reply:
we shouldn't do this, operator may not have any extra args needed for the compute